### PR TITLE
Saving and restoring of the number of active audio track

### PR DESCRIPTION
**Pull request by chshkhr**
Hi,

I like your application very much. Great job!

I need to open several times the same video file on the same audio track which is not the first in the list. 
Any time I had to switch it back manually...
I do not have any experience in C# but managed to solve the task by following your code.
I think it could be a useful new feature.

By the way, I use the application for studying foreign languages combining it with QTranslate. It's rather convenient but could be even better with something like keystroke macro.

### CODE REVIEW

**Comment by niksedk** [comment]
Cool, it works :)

File: src/libse/Common/Settings.cs [hunk 1]
```diff
@@ -2830,6 +2832,12 @@ private static Settings CustomDeserialize(string fileName)
                     videoFileName = listNode.Attributes["VideoFileName"].Value;
                 }
 
+                string audioTrack = "1";
```
Comment (niksedk) [inline src/libse/Common/Settings.cs:2835]: Could you used "-1" ?

File: src/ui/Forms/Main.cs [hunk 2]
```diff
@@ -22536,6 +22544,18 @@ private void GotoSubPositionAndPause(double adjustSeconds)
...
+                   if (mediaPlayer.VideoPlayer is LibVlcDynamic libVlc)
+                    {
+                        libVlc.AudioTrackNumber = VideoAudioTrackNumber;
+                    }
+                    else if (mediaPlayer.VideoPlayer is LibMpvDynamic libMpv)
+                    {
+                        libMpv.AudioTrackNumber = VideoAudioTrackNumber;
+                    }
+                }
+
```
Comment (niksedk) [inline src/ui/Forms/Main.cs:22558]: Could you move the above code to "VideoLoaded" ?

File: src/ui/Forms/Main.cs [hunk 3]
```diff
@@ -2820,7 +2820,7 @@ public double CurrentFrameRate
 
         private void OpenSubtitle(string fileName, Encoding encoding)
         {
-            OpenSubtitle(fileName, encoding, null, null);
+            OpenSubtitle(fileName, encoding, null, 1, null);
```
[the remark this answers is not in the document]
Comment (chshkhr) [reply src/ui/Forms/Main.cs:2823]: Done

**Pull request merged**